examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -46,6 +48,8 @@ teleportVersionOverride: ""
 # When set to true, the agent will skip the verification of proxy TLS
 # certificate.
 insecureSkipProxyTLSVerify: false
+# Enable DEBUG logging.
```

Comment:
let's not add debug log or any additional flags to helm chart. Our strategy is to allow users to create their own configs if they need override.

Reply:
sure.